packages/ethereum-contracts/contracts/agreements/ConstantFlowAgreementV1.sol

```diff
@@ -447,6 +452,23 @@ contract ConstantFlowAgreementV1 is
         }
 
         _requireAvailableBalance(flowVars.token, flowVars.sender, currentContext);
+
+        // @note It is possible this silently fails due to out of gas reasons, and users should
+        // still be able to recreate the hook behavior. This logic should exist in the NFT contract though.
```

Comment:
oh no :) now you used the N words! It's supposed to be a surprise!

Reply:
🐰🥚